### PR TITLE
Create blog post for KubeVirt RunStrategies

### PR DESCRIPTION
**Pull request by stu-gott**
Signed-off-by: Stu Gott <stu-gott@users.noreply.github.com>

**What this PR does / why we need it**:

Discusses the various RunStrategies available in KubeVirt

### CODE REVIEW

**Comment by stu-gott** [comment]
@codificat it's not clear to me what went wrong with Travis. Can you advise?

**Comment by codificat** [comment]
> @codificat it's not clear to me what went wrong with Travis. Can you advise?

It's the spell checker that is complaining because it doesn't understand [a few words](https://travis-ci.org/github/kubevirt/kubevirt.github.io/builds/738403446#L381-L391).

Some ot them you will have to add them to the dictionary in the `.yaspeller.json` file in the TLD (e.g. _RunStrategies_, the acronyms, your name). There is at least one legitimate typo (_accommodate_), and for a few of them you might consider actually changing the word, or add to the dictionary. I'm thinking in e,g, _October_ vs _Oct_  or _re-spawn_ vs _respawn_ .... up to you.

**Comment by codificat** [comment]
The current check failure is unrelated to this PR, it's a couple of broken links in an older blog post... once #646 merges, a rebase should let this pass.

**Comment by stu-gott** [comment]
Sounds good. I estimated a release date of tomorrow for this blog post to go live. Should I update the post date to be sometime next week then?

**Comment by stu-gott** [comment]
/cc @rmohr @davidvossel 

Would one of you mind reviewing this from a technical content perspective?

**Comment by codificat** [comment]
> Sounds good. I estimated a release date of tomorrow for this blog post to go live. Should I update the post date to be sometime next week then?

Well, we are already in _next week_, so that might be a sensible thing to do. If you update the post date, keep in mind that you should also rename the file to match it.

About the content: I'll let more knowledgeable engineers provide their review, but from my side I'll mention that I believe if would be nice to add a few relevant links within the text where appropriate - not as separate references, but in mentions. For example: link to the API spec when you mention it, the documentation about the new `RunStrategy` field...

By the way, a quick note on formatting: we usually use `code` format for references like `RunStrategy` or even `Running` (when it's a value in the spec/status), so I'd suggest to replace a few double quotes (") with backticks.

File: _posts/2020-10-30-run_strategies.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# High Availability
+
+No discussion of RunStrategies is complete without mentioning High Availability. After all, the implication behind the RerunOnFailure and Always RunStrategies is that your VM should always be available. For the most part this is completely true, but there's one important scenario where there's a gap to be aware of: if a node fails completely, e.g. loss of networking or power. Without some means of automatic detection that the node is no longer active, KubeVirt won't know that the VM has failed. On OpenShift clusters installed using Installer Provisioned Infrastructure (IPI) with MachineHealthCheck enabled can detect failed nodes and reschedule workloads running there.
+
+Mode information on IPI and MHC can be found here:
+
+[Installer Provisioned Infrastructure](https://cnv3520--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-overview.html#ipi-install-overview)
+[Machine Health Check](https://cnv3520--ocpdocs.netlify.app/openshift-enterprise/latest/machine_management/deploying-machine-health-checks.html#machine-health-checks-about_deploying-machine-health-checks)
```
Comment (mazzystr) [inline _posts/2020-10-30-run_strategies.md:52]: These Netlify links seem brittle.  Is there anyway to change to an official access.redhat.com link?

**Review by mazzystr — CHANGES_REQUESTED** [review]

File: _posts/2020-10-30-run_strategies.md (hunk 1 above)
Comment (stu-gott) [reply _posts/2020-10-30-run_strategies.md]: Fixed

**Comment by stu-gott** [comment]
@codificat Rebased and re-set the release date for Friday Dec 4. I think this post has had plenty of soak time at this point.

**Comment by codificat** [comment]
There was a broken link, I added a commit to fix it.

Besides that, for some reason the link checker was reporting link failures to the katacoda scenarios - maybe this was a temporary glitch, let's see if that passes now.

Other than that,
/lgtm
/approve

**Comment by codificat** [comment]
The link checker reports a failure again with the katacoda links (403, forbidden), but they do work. I'm wondering if it might be that katacoda doesn't like the user agent or something...

@mazzystr as you have been fighting with the link checker recently, does that ring a bell?

**Comment by stu-gott** [comment]
@codificat this sounds like something unrelated to this PR directly and an issue with existing links elsewhere in the repository?

**Comment by codificat** [comment]
> @codificat this sounds like something unrelated to this PR directly and an issue with existing links elsewhere in the repository?

Well, it's worse than that: tests are currently failing due to non-existing issues :confused: 

I'll merge this manually.

**Pull request merged**